ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2572,8 +2572,8 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
-- as: gcp-ipi-private-f14-disasterrecovery
-  cron: 27 19 2,17 * *
+- as: gcp-ipi-private-f6-disasterrecovery
+  cron: 27 19 5,10,17,23,29 * *
```

Comment:
I can see some duplicate date there. Is it expected ?
```
  cron: 11 11 1,7,13,19,25 * *
  cron: 11 11 2,8,14,20,26 * *
  cron: 35 19 3,9,15,21,27 * *
  cron: 35 19 4,10,16,22,28 * *
  cron: 27 19 5,10,17,23,29 * *
  cron: 27 19 6,12,18,24,29 * *
```

Reply:
Yes, GOOD CATCH! there are two duplicated date, updated it,